install/operator/pkg/apis/syndesis/v1alpha1/syndesis_types.go

```diff
@@ -91,16 +91,16 @@ type Resources struct {
 }
 
 type ResourcesWithVolume struct {
-	v1.ResourceRequirements `json:",inline,omitempty"`
-	VolumeCapacity          string `json:"volumeCapacity,omitempty"`
+	Resources      `json:",inline,omitempty"`
```

Comment:
Do we need to do [a upgrade procedure](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning/#upgrade-existing-objects-to-a-new-stored-version) for this (whatever that is?)

Reply:
This change does not affect the compatibility of json/encoding decoding, but it does allow common functions to work with with Resources or ResourcesWithVolume more easily.